packages/opencensus-web-all/src/export-initial-load.ts

```diff
@@ -66,7 +70,9 @@ export function exportRootSpanAfterLoadEvent() {
 function exportInitialLoadSpans() {
   setTimeout(() => {
     const spanContext = getInitialLoadSpanContext();
-    if (!isSampled(spanContext)) return; // Don't export if not sampled.
+    // Set the span context to allow passing it around OpenCensus Web.
+    setInitialLoadSpanContext(spanContext);
+    if (!isSampled()) return; // Don't export if initial load is not sampled.
```

Comment:
Would it make sense to pass the `spanContext` argument here so that it's clear to the reader that the sampling decision is coming from the initial load span context?